ftp/src/main/scala/akka/stream/alpakka/ftp/javadsl/FtpApi.scala

```diff
@@ -149,7 +151,22 @@ sealed trait FtpApi[FtpClient] { _: FtpSourceFactory[FtpClient] =>
       chunkSize: Int = DefaultChunkSize
   ): Source[ByteString, CompletionStage[IOResult]] = {
     import scala.compat.java8.FutureConverters._
-    ScalaSource.fromGraph(createIOGraph(path, connectionSettings, chunkSize)).mapMaterializedValue(_.toJava).asJava
+    ScalaSource.fromGraph(createIOSource(path, connectionSettings, chunkSize)).mapMaterializedValue(_.toJava).asJava
+  }
+
+  /**
+   * Java API: creates a [[Sink]] of [[ByteString]] to some file [[Path]].
+   *
+   * @param path the file path
+   * @param connectionSettings connection settings
+   * @return A [[Sink]] of [[ByteString]] that materializes to a [[CompletionStage]] of [[IOResult]]
+   */
+  def toPath(
+      path: Path,
```

Comment:
Is `java.nio.Path` really a good type to use here? The parameter refers to the path on the FTP server, while `Path` instances always has a `FileSystem` (which does not belong to the FTP server). Won't that mean that `path.toAbsolutePath.toString` likely will contain "C:\my\path" on windows for example?

I think it should just be a string, or a domain specific path-like type.